freesound/local_settings.example.py

```diff
@@ -104,3 +105,10 @@
 RAVEN_CONFIG = {
     'dsn': 'DSN_PROVIDED_BY_SENTRY',
 }
+
+# AWS
+
+AWS_ACCESS_KEY_ID = ''
+AWS_SECRET_ACCESS_KEY = ''
+AWS_SQS_QUEUE_URL = ''
+AWS_SQS_MESSAGES_PER_CALL = 1
```

Comment:
These parameters are not used if set to empty right?
Why not moving them to `settings.py` so that these are not needed to be manually copied to `local_settings.py` (and in production we override them normally)?

Reply:
done here https://github.com/MTG/freesound/pull/1172/commits/2a1b5b03cc9087c1db58caddc2eaabba44ed3099